server/apps/distributed-app/docs/modules/ROOT/pages/operate/metrics.adoc

```diff
@@ -14,7 +14,7 @@ We also support displaying them via https://grafana.com/[Grafana]. Two methods c
  
 == Expose metrics for Prometheus collection
 
-To enable James metrics, add ``extensions.routes`` to https://github.com/apache/james-project/blob/master/docs/modules/servers/pages/distributed/configure/webadmin.adoc[webadmin.properties] file:
+To enable James metrics, add ``extensions.routes`` to https://github.com/apache/james-project/blob/b9bebce005230c87d3b9e87e664709266bcd48c9/server/apps/distributed-app/docs/modules/ROOT/pages/configure/webadmin.adoc[webadmin.properties] file:
```

Comment:
Q: Can we point to the master branch? Otherwise, the latest doc changes can not be reflected.

Reply:
`b9bebce005230c87d3b9e87e664709266bcd48c9` is master now
Using "fixed" hash will help us avoid getting the "not found" in future

Reply:
After thinking, the `master` will be better for the document that often update